app/src/test/java/com/antosik/benchproject/app/common/binding/recyclerView/RecyclerViewBindingAdaptersKtTest.kt

```diff
@@ -0,0 +1,32 @@
+package com.antosik.benchproject.app.common.binding.recyclerView
+
+import androidx.recyclerview.widget.RecyclerView
+import com.antosik.benchproject.app.common.recyclerView.RecyclerBaseAdapter
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import org.junit.jupiter.api.Test
+
+internal class RecyclerViewBindingAdaptersKtTest {
+    val recyclerView: RecyclerView = mockk()
+    val adapter: RecyclerBaseAdapter<Any> = mockk(relaxed = true)
+
+    @Test
+    fun `SHOULD invoke method uploadData WHEN input data is NOT null`() {
+        val data: List<Any> = mockk(relaxed = true)
+        every { recyclerView.adapter } returns adapter
+
+        recyclerView.updateAdapterData(data)
+
+        verify { adapter.uploadData(data) }
+    }
+
+    @Test
+    fun `SHOULD don't invoke method uploadData WHEN input data is NOT null`() {
```

Comment:
Wrong name `WHEN input data is NOT null` but actually it's null

Reply:
Yes, it's my mistake I just copy it from other test :D

Reply:
Done :D